setup.cfg

```diff
@@ -1,5 +1,81 @@
+[metadata]
+name = pylint
+version = attr: pylint.__version__
+description = python code static checker
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+url = https://github.com/PyCQA/pylint
+author = Python Code Quality Authority
+author_email = code-quality@python.org
+license = GPL-2.0-or-later
+license_files =
+    COPYING
+    CONTRIBUTORS.txt
+classifiers =
+    Development Status :: 6 - Mature
+    Environment :: Console
+    Intended Audience :: Developers
+    License :: OSI Approved :: GNU General Public License (GPL)
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+    Topic :: Software Development :: Debuggers
+    Topic :: Software Development :: Quality Assurance
+    Topic :: Software Development :: Testing
+keywords = static code analysis linter python lint
+project_urls =
+    "What's New" = https://pylint.pycqa.org/en/latest/whatsnew/
+    "Bug tracker" = https://github.com/PyCQA/pylint/issues
+    "Discord server" = https://discord.gg/kFebW799
+    "User manual" = http://pylint.pycqa.org/en/latest/
+    "Contributing" = http://pylint.pycqa.org/en/latest/development_guide/contribute.html
+    "Technical references" = http://pylint.pycqa.org/en/latest/technical_reference/index.html
+    "Mailing list" = mailto:code-quality@python.org
+
+[options]
+packages = find:
+install_requires =
+    astroid>=2.5.2,<2.7
+    isort>=4.2.5,<6
+    mccabe>=0.6,<0.7
+    toml>=0.7.1
+    colorama;sys_platform=="win32"
+python_requires = ~=3.6
+zip_safe = True
```

Comment:
I'm not really sure how this settings works, which is why I tend to avoid it and just leave it out. AFAIK `setuptools` can automatically detect if the package is `zip_safe`. Thus this wouldn't be needed necessarily.

Reply:
setuptools is conservative before saying a package is zip_safe, that's why you could want to specify it. pylint performance problem probably do not stem from not being zipped though.

Reply:
👌🏻

Reply:
Apparently it's an old option that is not relevant for pip but was used for eggs. (So I removed it) See https://github.com/python/mypy/issues/8802#issuecomment-628249236